### ISSUE TITLE
TechCrunch search fails to load search results

### ISSUE THREAD

**Issue by justlurking** [EFForg/privacybadger#1579]
Go to https://techcrunch.com/search/ and search for a term with PB enabled - dozens of trackers are blocked and no search results are returned.  With PB disabled, search results are provided as expected.

I tried enabling various domains to see what was breaking the search functionality, but it seemed that as I enabled more domains even more new domains started popping up and being blocked.  I imagine someone with more experience will know how to diagnose this issue in a more efficient manner.

**Comment by ghostwords** [comment]
Does unblocking (toggling to "yellow" or "green") `api.swiftype.com` (and any other blocked `swiftype.com` domains) solve the problem?

**Comment by ghostwords** [comment]
Might be a duplicate of #1562.

**Comment by justlurking** [comment]
Yes!  That does indeed fix the issue, thank you.  I agree this does appear to be a duplicate of #1562.  Please let me know if I should do something to mark it as such.

**Comment by ghostwords** [comment]
OK cool, thanks for checking. All good, I'll close this one and take care of #1562 shortly.
